index.html

```diff
@@ -13,18 +13,21 @@
         <div class="col-md-4 col-md-offset-4">
           <h1 class="title">Weather App</h1>
           <form class="getWeatherData">
-            <input type="text" class="weather-city" placeholder="Enter a city.."/>
+            <input type="text" class="weather-city" name='q' placeholder="Enter a city..."/>
```

Comment:
what consistency of your quotes

Reply:
lol can you tell which style i prefer? i'll respect the file's default next time 👍